test/integration/golden_response/staging/get_triples/place_type_svobs.json

```diff
@@ -0,0 +1,5001 @@
+{
+  "Country": [
+    {
+      "subjectId": "wikidataId/Q756617",
+      "subjectName": "Kingdom of Denmark",
+      "subjectTypes": [
+        "Country"
+      ],
+      "predicate": "typeOf",
+      "objectId": "Country",
+      "objectName": "Country",
+      "objectTypes": [
+        "Class"
+      ],
+      "provenanceId": "dc/5j06ly1"
```

Comment:
Should we go back to the original state of the PR where the test files were shared?

Reply:
I reverted back and the diff looks good. I branched the test for get statvar triples, where the svobs mode adds "censusACSTableId" predicate.